src/main/java/edu/oakland/production/database/DatabasePersistentStorageClass.java

```diff
@@ -0,0 +1,167 @@
+package edu.oakland.production.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class DatabasePersistentStorageClass {
```

Comment:
Based on the UML Diagrams that we created, this should be called `DatabasePersistentStorageImplementation`, and should implement the `DatabasePersistentStorage` Java interface..
```suggestion
public class DatabasePersistentStorageImplementation implements DatabasePersistentStorage {
```